### DISCUSSION TITLE
Remote Sensing Applications to Support Locust Management and Research

### DISCUSSION THREAD

**Discussion by ajfriend** [uber/h3#860] [H3 Spotlight]
Remote Sensing Applications to Support Locust Management and Research
Evaluating the Potential of Earth Observation for Locust Outbreaks in Different Regions
Kumulative Dissertation zur Erlangung des Doktorgrades der Mathematisch-Naturwissenschaftlichen Fakultät der Christian-Albrechts-Universität zu Kiel
vorgelegt von Igor Klein

[2023-01-31_Dissertation-Igor-Klein.pdf](https://github.com/user-attachments/files/16215356/2023-01-31_Dissertation-Igor-Klein.pdf)

**Comment by aradfarahani** [comment]
https://github.com/aradfarahani/Remote-Sensing
check this out
